domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/stops/LocationBase.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.stops;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.GtfsEntity;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.Notice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Base class for all entities defined in stops.txt
+ */
+public abstract class LocationBase extends GtfsEntity {
+
+    @NotNull
+    public String getStopId() {
+        return stopId;
+    }
+
+    public String getStopCode() {
+        return stopCode;
+    }
+
+    public String getStopName() {
+        return stopName;
+    }
+
+    public String getStopDesc() {
+        return stopDesc;
+    }
+
+    public Float getStopLat() {
+        return stopLat;
+    }
+
+    public Float getStopLon() {
+        return stopLon;
+    }
+
+    public String getZoneId() {
+        return zoneId;
+    }
+
+    public String getStopUrl() {
+        return stopUrl;
+    }
+
+    public String getStopTimezone() {
+        return stopTimezone;
+    }
+
+
+    private @NotNull
+    final String stopId;
+    private final String stopCode;
+    private final String stopName;
+    private final String stopDesc;
+    private final Float stopLat;
+    private final Float stopLon;
+    private final String zoneId;
+    private final String stopUrl;
+    private final String stopTimezone;
+
+    protected LocationBase(@NotNull String stopId,
+                           String stopCode,
+                           String stopName,
+                           String stopDesc,
+                           Float stopLat,
+                           Float stopLon,
+                           String zoneId,
+                           String stopUrl,
+                           String stopTimezone) {
+        this.stopId = stopId;
+        this.stopCode = stopCode;
+        this.stopName = stopName;
+        this.stopDesc = stopDesc;
+        this.stopLat = stopLat;
+        this.stopLon = stopLon;
+        this.zoneId = zoneId;
+        this.stopUrl = stopUrl;
+        this.stopTimezone = stopTimezone;
```

Comment:
Is there a validation done on the timezone and the URL?

Reply:
Earlier in the process there are.

- URL have their format validated
- timezone are validated against a list of timezones packaged with the SDK (and that shall be updatable)